UKHomeOffice/sgmr-service
# SPCR-409 Update length check so the check doesn't fail

### AC / Description of changes
If there are 0 people then the variable savedPeople doesn't exist
And the code was failing to run .length when savedPeople doesn't exist

Updated syntax to make a stable check


### To Test
Have 0 people on your account
Create a voyage
The 'people' page should load with a prompt to save people


### Notes

This has been resolved in 2.1.0, however that is not due for release yet so this PR has a temporary fix to stop it breaking for now

---
* remember to merge to feature branches not master *
